app/controllers/users/sessions_controller.rb

```diff
@@ -17,7 +17,7 @@ def create
   rescue LoginIncompleteError
     render :login_email_sent
   rescue EmailNotFoundError
-    redirect_to :registrations_account_not_found
+    render :login_email_sent
```

Comment:
This doesn't look right. If the email doesn't exist I think depending on the user type they may have to contact support.

Reply:
This PR also changes the template for this action - it says something like
"If you have an account we sent you an email. You can check if you have an account under this link"

And checking the account is "contact the right people to know".

Reply:
The point is to make it so that you can't tell whether an account exists or not just by entering the email address